backend-rust/src/graphql_api/account_metrics.rs

```diff
@@ -90,12 +69,13 @@ impl QueryAccountMetrics {
         .await?;
 
         let x_time = rows.iter().map(|r| r.bucket_time).collect();
-        let y_last_cumulative_accounts_created =
-            rows.iter().map(|r| r.end_index.expect("coalesced")).collect();
-        let y_accounts_created = rows
-            .iter()
-            .map(|r| r.end_index.expect("coalesced") - r.start_index.expect("coalesced"))
-            .collect();
+        let y_last_cumulative_accounts_created: Vec<i64> =
+            rows.iter().map(|r| r.end_index).collect();
```

Comment:
Same goes for this. If nothing is returned this is zero which is obviously wrong. I do not know where this is used as of yet though.